enclave-runtime/Cargo.lock

```diff
@@ -1593,12 +1600,15 @@ dependencies = [
 name = "its-consensus-common"
 version = "0.8.0"
 dependencies = [
+ "itp-ocall-api",
  "itp-sgx-crypto",
  "itp-types",
  "its-primitives",
  "its-state",
+ "log 0.4.14 (git+https://github.com/mesalock-linux/log-sgx)",
```

Comment:
Didn't want to get away from the mesalock log, and only  use the rust's log, without default features?

Reply:
For most crates we did, but in the `enclave-runtime` we've still had it, and we've patched it in the `cargo.toml`. Maybe @haerdib knows more?

Reply:
Yes, we still have it in some crates as well as enclave-runtime as far as I know. I changed it in some .toml I've touched but not in all.. I guess that's an ongoing process.